src/components/maintabsmanager.js

```diff
@@ -166,6 +166,7 @@ define(['dom', 'browser', 'events', 'emby-tabs', 'emby-button'], function (dom,
             }).join('') + '</div></div>';
 
             tabsContainerElem.innerHTML = tabsHtml;
+            CustomElements.upgradeSubtree(tabsContainerElem);
```

Comment:
to calm eslint
```suggestion
            window.CustomElements.upgradeSubtree(tabsContainerElem);
```

Reply:
We can also add CustomElements to the globals list in .eslintrc.js, but using `window` is probably clearer.